xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/widgets/locationPicker.css

```diff
@@ -7,23 +7,47 @@
   /* Remove the margin so that the validation message is displayed right below the breadcrumb, for simple users.
      For advanced users the .location-edit element that follows the breadcrumb has a top margin. */
   margin-bottom: 0;
+  /*Overwrite style properties*/
+  background-color: transparent;
+  border-radius: 0px;
+  padding: 0;
+  /*Remove the ol left padding*/
+  padding-left: 0;
+}
+
+dd.location-picker-preview{
+  /*Make the location picker display the bg color*/
+  background-color: #f8f8f8;
```

Comment:
The color shouldn't be hardcoded, the background color is defined in the less variable `@breadcrumb-bg` (should be available in `$theme.breadcrumbBg` via Velocity). There are color themes that are adding a border to the breadcrumbs:

https://github.com/xwiki/xwiki-platform/blob/226762d964f8a4ba298d84a3bacad3eb42efed03/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-theme/xwiki-platform-flamingo-theme-bootswatch/src/main/resources/FlamingoThemes/Lumen.xml#L2129